test/CMakeLists.txt

```diff
@@ -161,10 +166,10 @@ foreach(test_src ${test_OUT_2_srcs})
   endforeach()
 endforeach()
 
-# OUT_3 and OUT_4 tests
+# OUT_3 and OUT_4 tests; these are for "DA" (dynamic allocation) builds only
 foreach(test_src IN ITEMS ${test_OUT_3_srcs} ${test_OUT_4_srcs})
   string(REPLACE ".f" "" testPref ${test_src})
-  foreach(kind ${test_kinds})
+  foreach(kind ${test_kinds_4})
```

Comment:
the if statement on the next line will never have 8 or d, so please update the logic

Reply:
True, but in the future I may add some additional tests of 8 and d builds, so I'd prefer to leave that line as is for now.